parts/k8s/rotate-certs.sh

```diff
@@ -37,6 +37,11 @@ cp_certs() {
 
 cp_proxy() {
   source /etc/environment
+  local NODE_INDEX
+  NODE_INDEX=$(hostname | tail -c 2)
+  if [[ $NODE_INDEX == 0 ]]; then
```

Comment:
so the idea is that only one control plane node will update the cert, and we are doing this operation serially so the remaining control plane nodes will perform a successful `etcdctl get`?

Reply:
Correct, it is serial and ordered (master-0 comes first). See the second commit for context/clarification.